spacy/tests/vocab_vectors/test_lexeme.py

```diff
@@ -1,7 +1,62 @@
-import pytest
 import numpy
+import pytest
 from spacy.attrs import IS_ALPHA, IS_DIGIT
+from spacy.lookups import Lookups
+from spacy.tokens import Doc
 from spacy.util import OOV_RANK
+from spacy.vocab import Vocab
+
+
+@pytest.mark.issue(361)
+@pytest.mark.parametrize("text1,text2", [("cat", "dog")])
+def test_issue361(en_vocab, text1, text2):
+    """Test Issue #361: Equality of lexemes"""
+    assert en_vocab[text1] == en_vocab[text1]
+    assert en_vocab[text1] != en_vocab[text2]
+
+
+@pytest.mark.issue(595)
+@pytest.mark.skip(reason="Old vocab-based lemmatization")
+def test_issue595():
+    """Test lemmatization of base forms"""
+    words = ["Do", "n't", "feed", "the", "dog"]
+    lookups = Lookups()
+    lookups.add_table("lemma_rules", {"verb": [["ed", "e"]]})
+    lookups.add_table("lemma_index", {"verb": {}})
+    lookups.add_table("lemma_exc", {"verb": {}})
+    vocab = Vocab()
+    doc = Doc(vocab, words=words)
+    doc[2].tag_ = "VB"
+    assert doc[2].text == "feed"
+    assert doc[2].lemma_ == "feed"
+
+
+@pytest.mark.issue(599)
```

Comment:
Quite unsure if this test should be in here

Reply:
This one feels more about serialization. What you can do is go check with the original issue. Ideally, the labels on the issue will help, if not, hopefully the title/description make it more clear what the issue is about.

Reply:
Resolved in https://github.com/explosion/spaCy/pull/9655/commits/626a571774a517e4b22d59463ef3d61fb36f836f